R/metrics-sample.R

```diff
@@ -494,30 +534,59 @@ mad_sample <- function(observed = NULL, predicted, ...) {
 #' real-time epidemic forecasts: A case study of Ebola in the Western Area
 #' region of Sierra Leone, 2014-15, \doi{10.1371/journal.pcbi.1006785}
 #' @keywords metric
-pit_sample <- function(observed,
-                       predicted,
-                       n_replicates = 100) {
+pit_histogram_sample <- function(observed,
```

Comment:
noting if/when we get rid of get_ then pit_histogram and pit_histogram_sample are quite hard to distinguish.